libraries/AP_AHRS/AP_AHRS_DCM.cpp

```diff
@@ -81,6 +84,13 @@ AP_AHRS_DCM::update(bool skip_ins_update)
     // Perform drift correction
     drift_correction(delta_t);
 
+    if(_ins.get_gyro().z>GYRO_MAX_SPIN_RATE*3.14/180.0 || _ins.get_gyro().z<-GYRO_MAX_SPIN_RATE*3.14/180.0)
```

Comment:
3.14/180... We have defines for these.. with much better precision

Reply:
got it, I will fix it